internal/util/test/webhooks.go

```diff
@@ -104,18 +109,42 @@ ctFsgXhf5+tDgbBZpcuTMpd3KnaDUYg=
 -----END PRIVATE KEY-----`
 )
 
+// XXX (this hack is tracked in https://github.com/Kong/kubernetes-ingress-controller/issues/1613):
```

Comment:
Can you please reformat the comment? e.g. remove the `XXX` prefix and make it more readable?

Reply:
Done my best: [4966da7](https://github.com/Kong/kubernetes-ingress-controller/pull/2925/commits/4966da7aa095c9260aea218e2ee7d240ef572c15)